raiden/tests/unit/test_transfer.py

```diff
@@ -337,3 +359,97 @@ def test_healthcheck_with_bad_peer(raiden_network):
     # Let's make sure no new pings are sent afterwards
     gevent.sleep(2)
     assert len(messages) == final_messages_num
+
+
+@pytest.mark.parametrize('blockchain_type', ['mock'])
+@pytest.mark.parametrize('number_of_nodes', [1])
+@pytest.mark.parametrize('channels_per_node', [0])
+def test_receive_directtransfer_unknown(raiden_network, private_keys):
+    app0 = raiden_network[0]  # pylint: disable=unbalanced-tuple-unpacking
+
+    a0_address = pex(app0.raiden.address)
+    asset_manager0 = app0.raiden.managers_by_asset_address.values()[0]
+
+    other_key = PrivateKey(sha3("rainyweather"), ctx=GLOBAL_CTX, raw=True)
+    other_address = privatekey_to_address(other_key.private_key)
+    direct_transfer = DirectTransfer(
+        identifier=1,
+        nonce=1,
+        asset=asset_manager0.asset_address,
+        transferred_amount=10,
+        recipient=a0_address,
+        locksroot=sha3("muchrain")
+    )
+    sign_and_send(direct_transfer, other_key, other_address, app0)
+
+
+@pytest.mark.parametrize('blockchain_type', ['mock'])
+@pytest.mark.parametrize('number_of_nodes', [1])
+@pytest.mark.parametrize('channels_per_node', [0])
+def test_receive_mediatedtransfer_unknown(raiden_network, private_keys):
+    app0 = raiden_network[0]  # pylint: disable=unbalanced-tuple-unpacking
+
+    a0_address = pex(app0.raiden.address)
+    asset_manager0 = app0.raiden.managers_by_asset_address.values()[0]
+
+    other_key = PrivateKey(sha3("rainstopped"), ctx=GLOBAL_CTX, raw=True)
+    other_address = privatekey_to_address(other_key.private_key)
+    amount = 10
+    locksroot = sha3("muchsunny")
+    mediated_transfer = MediatedTransfer(
+        identifier=1,
+        nonce=1,
+        asset=asset_manager0.asset_address,
+        transferred_amount=amount,
+        recipient=a0_address,
```

Comment:
@hackaugusto Apologies. I think that I saw this used someplace and thus blindly copied it. Looked at `pex()` and I see that this was mistaken. That's the only beef I got with untyped languages ;)

But why not just use `app0.raiden.address`. It's an ethereum address: https://github.com/raiden-network/raiden/blob/master/raiden/raiden_service.py#L87